lang/en/block_filtered_course_list.php

```diff
@@ -32,9 +32,9 @@
 $string['catrubrictpl']             = 'Category rubric template';
 $string['catseparator']             = 'Category separator';
 $string['completedcourses']         = 'Completed courses';
-$string['configcatrubrictpl']       = 'Use this setting to determine a pattern for displaying rubrics when filtering by category. The following replacement tokens are available: NAME, IDNUMBER, PARENT, ANCESTRY. <br /><br />Full details at <a href="https://github.com/CLAMP-IT/moodle-blocks_filtered_course_list/wiki">https://github.com/CLAMP-IT/moodle-blocks_filtered_course_list/wiki</a>.';
+$string['configcatrubrictpl']       = 'Use this setting to determine a pattern for displaying rubrics when filtering by category. The following replacement tokens are available: NAME, IDNUMBER, PARENT, ANCESTRY. TOKEN{#} will ellipsize the value to # characters. <br /><br />Full details at <a href="https://github.com/CLAMP-IT/moodle-blocks_filtered_course_list/wiki">https://github.com/CLAMP-IT/moodle-blocks_filtered_course_list/wiki</a>.';
```

Comment:
I know this is now fiddly, but let's spend a little time on the help string, because it will be fairly important. Building from your suggestion I might propose: 

> You can truncate the output of any token by appending a character length in curly braces. (e.g. NAME{20}). An ellipsis will be added.

Reply:
Sounds good to me.